alphagov/paas-datadog-bbs-integration
# [#139196683] BBS integration

## What

This is a Datadog integration with BBS.

It provides BBS check and a test for it. 

The check is connecting over HTTPS to BBS API and determining it's health. It provides two metrics:

- `cf.bbs.ResponseTime` is provided when the check can connect
- `cf.bbs.Healthy` is a binary metric to determine whether BBS is up (`1`) or down (`0`)

## How to test

- code sanity check
- deploy together with `paas-cf` PR: https://github.com/alphagov/paas-cf/pull/818

## Who

Not @mtekel or @combor
